src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -25,6 +25,148 @@ public struct Vector256<T> where T : struct
          private int _dummy;
     }
 }
+namespace System.Runtime.Intrinsics.Arm.Arm64
+{
+    public static class Simd
+    {
+        public static bool IsSupported { get { throw null; } }
+        public static Vector64<byte>    Abs(Vector64<sbyte>   value) { throw null; }
```

Comment:
nit: I don't think we are aligning the names/etc elsewhere. (CC. @eerhardt to make sure this is ok).

Reply:
This style isn't mentioned in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, so I wouldn't block on it.

But in general, my suggestion is to not use formatting that is going to fight with VS's auto-formatting features (`CTRL+K, CTRL+D`).  It just causes hassle for the next editor(s) of the file.

Reply:
I'll clean up whitespace in a separate PR.